docs/ipv6.md

```diff
@@ -1,114 +1,175 @@
-# Ipv6 support in kube-router
+# IPv6 / DualStack Support in kube-router
 
-This document describes the current status, the plan ahead and general
-thoughts about ipv6 support in `kube-router`.
+This document describes the current status, the plan ahead and general thoughts about IPv6 / Dual-Stack support in
+kube-router.
 
-Ipv6-only is supported (in alpha) in `Kubernetes` from version 1.9 and
-support for ipv4/ipv6 dual stack is on the way
-([KEP](https://github.com/leblancd/community/blob/dual-stack-kep/keps/sig-network/0013-20180612-ipv4-ipv6-dual-stack.md)). It
-is desirable for `kube-router` to keep up with that development.
+Dual-Stack (e.g. IPv4 and IPv6) has been supported in Kubernetes since version `v1.21`:
+[IPv4/IPv6 dual-stack documentation](https://kubernetes.io/docs/concepts/services-networking/dual-stack/)
 
-The idea is to implement ipv6-only function-by-function;
+kube-router's current approach is to implement dual-stack functionality function-by-function:
 
  * CNI `--enable-cni`
  * Proxy `--run-service-proxy`
  * Router `--run-router`
  * Network policies `--run-firewall`
 
-It is important to always keep dual-stack in mind. The code must not
-be altered to handle ipv6 instead of ipv4 but must be able to handle
-both at the same time in the near future.
 
-To use ipv6 is usually not so hard in golang. The same code is used,
-only the addresses differ. This is also true for `iptables` and
-`ipvsadm`. This makes support for ipv6 a bit easier to implement.
+## Current status (January 22, 2023)
 
-## Testing
+Support for dual-stack in kube-router is under active development and is currently the main focus of development for the
+kube-router maintainers. Currently, we are targeting a v2.0.0 release of kube-router which will see all controllers
+updated for dual-stack compatibility.
 
-Test and development has so far used
-https://github.com/Nordix/xcluster/tree/master/ovl/kube-router-ipv6
-which is an easy way to get a ipv6-only Kubernetes cluster.
+We are currently running this work off of the
+[prep-v2.0 branch](https://github.com/cloudnativelabs/kube-router/tree/prep-v2.0) and, as of the time of this writing,
+have released a [release candidate]() with this some dual-stack functionality built into it.
 
-To setup an ipv6-only Kubernetes cluster is usually no simple task,
-see for instance https://github.com/leblancd/kube-v6
+Functions that currently support dual-stack on the v2.0.0 release line:
 
-No automatic tests exist yet for ipv6.
+* CNI
+* Router / BGP (`--run-router`)
+* Network Policies (`--run-firewall`)
 
+## How Can You Help?
 
-## Current status (Thu Oct 11 2018)
+The work to make dual-stack support workable in kube-router has involved a lot of work and several large contributions
+to the code base. If you have access to a development or staging environment where you would be willing to run the
+latest v2.0.0 release candidate and give us feedback, it would be much appreciated!
```

Comment:
Maybe add how to provide feedback (e.g. slack, issue, etc.)

Reply:
Added in the paragraph below.